modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc

```diff
@@ -263,10 +263,12 @@ Alternatively, you can set your autoscaling preferences for the default machine
 
 . Optional: Select an EC2 Instance Metadata Service (IMDS) configuration - `optional` (default) or `required` - to enforce use of IMDSv2. For more information regarding IMDS, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html[Instance metadata and user data] in the AWS documentation.
```

Comment:
This is interesting. Do we here intend to just document the defaults or also provide what other options are supported? For example, for auto-scaling (default No) or encryption (no KMS based encryption), we don't provide other options supported.

Reply:
I based these updates on what we currently have, but we can assess the cluster creation docs to see what needs to be added as an overall improvement effort.

Reply:
I agree with this, I feel like we should remove this box. I think this goes to show that we should likely have an article on how to deploy HCP "with customizations" as we do in other cases.

Reply:
Do you think this works as a stop-gap and then we can roll out a procedure for HCP with customizations? Or would you rather have it removed entirely and we can write the HCP with customizations in a follow-up PR?

Reply:
I think we can just leave it off and then do the HCP with customizations as a follow-up PR.

Reply:
I'd leave these boxes off of this table that lists the default. It defocuses what this table/page intends to do.

Reply:
This makes sense. I opened an issue to document "Creating ROSA with HCP clusters with customizations": https://issues.redhat.com/browse/OSDOCS-11730